.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+### Prerequisites
+
+- [ ] Have you written a descriptive issue title?
+- [ ] Have you verified that you are running the latest version of Nancy?
+- [ ] Have you verified if the problem exist in both `debug` and `release` mode?
+- [ ] Have you searched [open issues](https://github.com/NancyFx/Nancy/issues) to ensure it has not already been reported?
+
+### Description
+[A description of the bug or feature]

```

Comment:
> So the idea would be that it showed up in edit-mode, but was hidden once it was posted? I see your point, but I don't think we should make them optional in that way

Yes, that's the point. Optional? What makes them optional by doing it this way? The only thing it does is make sure that it's not echoed back to us as if the issue/PR author wrote it :stuck_out_tongue_closed_eyes: I'm not saying we comment out *everything*, just the placeholders.

> The whole point of the templates is to try and enforce a higher standard of the reported issues. If we call failure on it before we've even tried, it then what's the point? Also, if we have a template, and people fail to fill it out, we'll simply ask them to go back and update the description or the issue will be closed

I'm not "calling failure" on it, I'm simply saying that it's inevitable that a lot of people will fail to fill out everything. Either because it doesn't apply to them, or they're just lazy. I'd just like to keep things that are only targeted at the issue/PR author to *not* show up in the final issue, like "Thanks for reporting the issue to Nancy!".